aws/provider.go

```diff
@@ -427,6 +427,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_gamelift_alias":                               resourceAwsGameliftAlias(),
 			"aws_gamelift_build":                               resourceAwsGameliftBuild(),
 			"aws_gamelift_fleet":                               resourceAwsGameliftFleet(),
+			"aws_gamelift_queue":                               resourceAwsGameliftQueue(),
```

Comment:
The Gamelift API refers to these as `GameSessionQueue` -- we should likely follow their convention in case they introduce other types of queues. To update, `aws_gamelift_game_session_queue` for here and file names along with updated function names `resourceAwsGameliftGameSessionQueue`, etc.

This new resource should also have documentation added:
* A new page added: `website/docs/r/gamelift_game_session_queue.html.markdown`
* A link in the sidebar added: `website/aws.erb`